File: Day 1/Task 1/Task_1.py
fhandle = open('day_1_1.txt')

calories = 0
max_calories = 0

for line in fhandle:

	line = line.rstrip()
	
	if line != '':
		calories += int(line)
	else:
		if calories > max_calories:
			max_calories = calories

		calories = 0

fhandle.close()
print(f"The maximum total number of calories an elf is carrying is {max_calories}")

File: Day 1/Task 1/Task 1.cpp
#include <iostream>
#include <fstream>
#include <sstream>
#include <string>

int main()
{
	std::fstream fhandle("day_1_1.txt", std::ios::in);
	
	int max_calories = 0, calories = 0;
	int i = 0;
	std::stringstream buffer;
	std::string str;

	while (true) {
		std::getline(fhandle, str);
		if (fhandle && str != "") {
			buffer << str;
			int num;
			buffer >> num;
			buffer.ignore(10000, '\n');
			buffer.clear();
			calories += num;
		}
		else {
			if (calories > max_calories)
				max_calories = calories;
			calories = 0;

			if (!fhandle)
				break;
		}
	}
	std::cout << "The maximum total number of calories an elf is carrying is " << max_calories;
	
	
}

